io.github.jeffshee.Hidamari.json

```diff
@@ -0,0 +1,240 @@
+{
+    "app-id": "io.github.jeffshee.Hidamari",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "hidamari",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--filesystem=xdg-config/hidamari",
+        "--filesystem=xdg-config/autostart",
```

Comment:
or this?

Reply:
The app needs to write a `.desktop` file into that directory if the user wants the app to autostart after login.